Postman/Survey/SurveySample.postman_collection.json

```diff
@@ -0,0 +1,361 @@
+{
```

Comment:
Why is this indentation so different?

Reply:
I don't know. I'll try to unify it in the next task.